app/helpers/portal_helper.rb

```diff
@@ -33,4 +33,8 @@ def get_theme_icon(theme)
       'icons/monitor'
     end
   end
+
+  def generate_gradient_to_bottom(theme)
```

Comment:
Please add the spec for this method, `generate_gradient_to_bottom`

Reply:
Done

Reply:
Done